notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -18,6 +18,53 @@ https://en.wikipedia.org/wiki/List_of_science_fiction_films_of_the_2000s
 Collection: Science fiction films of the 2000s
 
 
+movie: Supernova (2000)
+=======================
+
+:: Title
+Supernova
+
+:: Description
+Supernova is a 2000 science fiction horror film written by David C. Wilson,
+William Malone and Daniel Chuba and directed by Walter Hill, credited as
+"Thomas Lee."
+
+Synopsis: A search and rescue medical ship crew in deep space discovers a
+potentially dangerous alien artifact form after responding to a distress
+signal.
+
+:: Date
+2000-01-14
+
+:: References
+https://en.wikipedia.org/wiki/Supernova_(2000_film)
+
+:: Ratings
+2 <paul>
+
+:: Major Themes
+sun going supernova [The story culminated with the revelation that the nine-dimensional matter inside the alien artifact would make the blue giant star go supernova, and moreover that the resulting explosion would reach Earth within 51 years. This, the ship computer hypothesized, would either destroy life on Earth or "enable humankind to achieve a new level of existence".]
+speculative spaceship [The film is largely set aboard the 22nd century, faster-than-light traveling medical rescue vessel Nightengale 229. Another novel aspect of the ship is that it used rotation to generate gravity.]
```

Comment:
Rotation to generate gravity is an interesting form of artificial gravity we could invent a theme for

Reply:
Indeed. Proposed the new theme **rotational simulated gravity** and added it as minor here.